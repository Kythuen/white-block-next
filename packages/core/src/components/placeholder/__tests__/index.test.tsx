import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import Placeholder from '../placeholder.vue'
// import { usePlaceholder } from '../use'

describe('# Placeholder', () => {
  describe('## Base', () => {
    it('### default config', () => {
      const wrapper = mount({
        render() {
          return <Placeholder />
        }
      })
      // expect(wrapper.classes()).toEqual(['wb-placeholder'])
      // expect(wrapper.props()).toEqual({})
      expect(wrapper.element).toMatchSnapshot()
    })
  })

  describe.skip('## Slots', () => {})

  describe.skip('## Events', () => {})

  describe.skip('## Props', () => {})

  describe.skip('## Composables', () => {})
})
