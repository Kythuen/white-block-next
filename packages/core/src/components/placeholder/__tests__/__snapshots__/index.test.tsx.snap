// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`# Placeholder > ## Base > ### default config 1`] = `
<div
  class="wb-placeholder wb-placeholder--line wb-placeholder--md"
  flex="~ row"
  h="full"
  items="center"
  justify="center"
  overflow="hidden"
  ring="1 $wb-placeholder-border"
  rounded="$wb-placeholder-radius"
  w="full"
>
  <div
    max-h="85%"
    max-w="85%"
    overflow="hidden"
    p="x-$wb-placeholder-padding"
    select="none"
  >
    <!---->
  </div>
</div>
`;
